templates/containers/what-are-containers.html

```diff
@@ -0,0 +1,243 @@
+
+{% block meta_image %}https://assets.ubuntu.com/v1/fece8a11-container-in-the-cloud.svg{% endblock meta_image%}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1hStuIzY8dDdFEW4Q0TwboOx0_-SB5klk8zRuNIn3V4o/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+    <div class="row u-equal-height">
+      <div class="col-7">
+        <h1>What are containers?</h1>
+        <p>Containers are a virtualisation technology that isolates each process in a lightweight and standard environment. They allow users to package and quickly deploy entire applications parallel to each other on the same kernel and hardware, while maintaining isolation among the workloads.</p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/fece8a11-container-in-the-cloud.svg",
+          alt="",
+          width="200",
+          height="163",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Container virtualization</h2>
+    <p>Containers are instantiated from container images, a layered packaging system, to distribute applications along with their configuration and runtime dependencies.</p>
+    <p>From development to production, containers power modern cloud applications as they allow developers to move fast between platforms and versions.</p>
+    <p>Inside Google alone, at least <a href="https://cloud.google.com/containers/" class="p-link--external">two billion containers are generated each week</a> to manage its vast operations. There are many tools to create, deploy and manage containers, including LXD, Docker and Kubernetes.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>What is Docker?</h2>
+    <p><a href="https://www.docker.com/" class="p-link--external">Docker</a> is an open-source containerisation platform. It provides an easy way to build and deploy containers on the cloud or on-premises. Container technologies were not introduced by Docker, but they made the user experience of container technologies so easy for developers that Docker soon became shorthand for containers as well as the most popular container format. <a href="https://www.docker.com/" class="p-link--external">Docker Inc</a>, the company that supports the Docker project and community, also operates <a href="https://hub.docker.com/" class="p-link--external">Docker Hub</a>, a space for hosting container images.</p>
+  </div>
+</section>
+
```

Comment:
There is a missing section - "Docker for beginners: docker-cli, containerd, and runc"  here between "What is Docker?" and "Kubernetes vs. Docker: CRI-O and OCI
". "Docker for beginners: docker-cli, containerd, and runc"